src/protobuf.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _PROTOBUF_H
+#define _PROTOBUF_H
+
+#include <sys/param.h>
+#include <string.h>
+#include <stdlib.h>
```

Comment:
It's generally preferable to keep number of #includes minimal in header files (this may improve compilation speed when sources are not merged).
In this case, all these standard headers can be relocated to .cpp. Implementation of ProtoBuffer constructor/destructor will need to move to .cpp as well, but that's fine.

Reply:
eeca655089c0818afbf61695210789c79c1918d6